src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -622,60 +622,77 @@ PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<Tar
 {
   // TODO replace with call to distributable function
 
-  const int min_segment_num = this->proj_data_info_sptr->get_min_segment_num();
-  const int max_segment_num = this->proj_data_info_sptr->get_max_segment_num();
-
   info(boost::format("Calculating sensitivity for subset %1%") % subset_num);
 
   int min_timing_pos_num = use_tofsens ? this->proj_data_info_sptr->get_min_tof_pos_num() : 0;
   int max_timing_pos_num = use_tofsens ? this->proj_data_info_sptr->get_max_tof_pos_num() : 0;
-  if (min_timing_pos_num < 0 || max_timing_pos_num > 0)
+  if (min_timing_pos_num < 0 || max_timing_pos_num > 1)
```

Comment:
```suggestion
  if (min_timing_pos_num < 0 || max_timing_pos_num > 0)
```

Reply:
No, this is correct. Up to one TOF bin should not be considered TOF reconstruction.

Reply:
still not with you on this one (i.e. with the 1). Why not say `num_timing_poss > 1`